### PR TITLE
Make jenkins imagestream

### PR DESCRIPTION
**Pull request by jflowers**
Added tasks to create imagestream and tags as required by the OCP Jenkins Template in the openshift namespace
collecting creds for the disconnected registry with a var_prompt, please be critical on how this was done

### CODE REVIEW

File: roles/legacy-jenkins/tasks/main.yml [hunk 1]
```diff
@@ -14,6 +14,32 @@
 # https://docs.openshift.com/container-platform/4.5/openshift_images/using_images/images-other-jenkins.html#images-other-jenkins-env-var_images-other-jenkins
 # looks like the env var OVERRIDE_PV_PLUGINS_WITH_IMAGE_PLUGINS would mean that we don't have to worry as long as the default plugins are sufficient 
 
+- name: Ensure Jenkins controller ImageStream exists in disconnected cluster
+  when: not connected_cluster
+  block:
+
+  - name: List openshift-release-dev image tags
+    uri:
+      url: 'https://{{ rhedsord_registry }}/v2/openshift-release-dev/tags/list'
```
Comment (johannes-cabal) [inline roles/legacy-jenkins/tasks/main.yml]: Extract `openshift-release-dev` as a variable. Tho this shouldn't change, want to be able to modify outside of a task file.

File: roles/legacy-jenkins/tasks/main.yml [hunk 2]
```diff
@@ -14,6 +14,32 @@
...
+      method: GET
+      user: '{{ rhedsord_registry_username }}'
+      password: '{{ rhedsord_registry_password }}'
+      force_basic_auth: yes
+      status_code: 200
+      validate_certs: false
+    register: openshift_release_dev_image_tags
+
+  - set_fact:
+      jenkins_image_path: '{{ rhedsord_registry }}/openshift-release-dev:{{ item }}'
```
Comment (johannes-cabal) [inline roles/legacy-jenkins/tasks/main.yml]: Extract openshift-release-dev as a variable. Tho this shouldn't change, want to be able to modify outside of a task file.

**Comment by johannes-cabal** [comment]
@jflowers  please see comments. Just need to extract the repository name and we're good-to-go. Thanks! 

Also added a link to this in the respective JIRA task.

**Comment by jflowers** [comment]
Added var for the repo name

**Review by johannes-cabal — APPROVED** [review]

**Pull request merged**